lib/ansible/utils/path.py

```diff
@@ -43,6 +43,9 @@ def unfrackpath(path, follow=True):
         '$HOME/../../var/mail' becomes '/var/spool/mail'
     '''
 
+    if path == '-':
+        return path
```

Comment:
I'm not sure if this should be in unfrack path, it is used in many locations that will not work with stdin/stdout.
A safer fix is probably doing this in the __cli__ options and short circuit calling `unfrackpath` if path == '-'

Reply:
Right, I thought about this as well, but I'm not sure you ever want to change `-` so I went with my gut and added it at the lowest possible common denominator. Are you certain moving it up is the right way?

Reply:
yes, the only context  this should be a alias for the 'std' file descriptors should be on the CLI options